src/reflow/mod.rs

```diff
@@ -59,27 +57,20 @@ impl Checker for Reflow {
                         });
                     // remove last newline
                     let _ = new_lines.pop();
-                    // @todo that's too easy
-                    if new_lines.len() == chunk.as_str().lines().count() {
+                    // no reflow requried
+                    // @todo still looks quite fishy, rethink those replace() calls
+                    if new_lines.join("") == chunk.as_str().replace("\n", "").replace("   ", " ") {
                         log::trace!("No rewrapping required for '{:?}'", chunk);
                         continue; // the chunk did not change
                     }
-                    // @todo find proper span and range
-                    let range = 0..chunk.as_str().len();
-                    let chunk_it = chunk.iter().next();
-                    let (range_it, span) = chunk_it.unwrap();
-                    let mut range_it = range_it.clone();
-                    let mut span = span.clone();
-                    for (r, s) in chunk_it {
-                        if range_it.start > r.start {
-                            range_it.start = r.start;
-                            span.start = s.start;
-                        }
-                        if range_it.end < r.end {
-                            range_it.end = r.end;
-                            span.end = s.end;
-                        }
-                    }
+                    // find proper span and range
+                    let (first_range, first_span) = chunk.first_span().unwrap();
```

Comment:
Might be worth to move this to a new fn in `CheckableChunk` , `fn complete_coverage()`

Reply:
sounds reasonable, will do!